src/components/CreateShareStructure/ListShareClass.vue

```diff
@@ -1,14 +1,368 @@
 <template>
-<div></div>
+  <v-card flat id="share-structure">
+
+    <!-- Summary Section -->
+    <div id="share-summary" v-if="isSummary">
+      <!-- Summary Header -->
+      <div class="share-summary-header" >
+        <v-icon>mdi-file-tree</v-icon>
+        <label class="share-summary-header-title"><strong> Share Structure</strong></label>
+      </div>
+
+      <!-- Summary Warning -->
+      <div v-if="showErrorSummary" class="share-summary-invalid-message">
+        <span>
+          <v-icon color="#1976d2">mdi-information-outline</v-icon>
+          This step is not complete.
+          <router-link id="router-link" :to="{ path: '/create-share-structure', query: { showErrors: true } }">
+            Return to this step to complete it.
+          </router-link>
+        </span>
+      </div>
+    </div>
+
+    <v-data-table
+      :headers="headers"
+      :items="shareClasses"
+      :disable-pagination="true"
+      :disable-sort="true"
+      :hide-default-footer="true"
+    >
+      <template v-slot:item="row" class="share-data-table">
+
+        <!-- Share Class Rows-->
+        <tr :key="row.item.id" class="class-row" :class="{ 'class-row-has-series': row.item.series.length}">
+          <td class="list-item__title">{{row.item.name}}</td>
+          <td>{{row.item.maxNumberOfShares || 'No Maximum'}}</td>
+          <td>{{row.item.parValue ? `$${row.item.parValue}` : 'No Par Value'}}</td>
+          <td>{{row.item.currency}}</td>
+          <td>{{row.item.hasRightsOrRestrictions ? 'Yes' : 'No'}}</td>
+
+          <!-- Share Class Edit Btn -->
+          <td v-if="!isSummary">
+            <div class="actions">
+              <span class="edit-action">
+                <v-btn small text color="primary"
+                 :id="'Class-' + row.index + '-change-btn'"
```

Comment:
IDs should be in lower-case kebab-case, eg "abc-def".

Reply:
Fixed